scripts/run_backend_tests.py

```diff
@@ -343,15 +341,10 @@ def main(args=None):
     dev_appserver.fix_sys_path()
 
     if parsed_args.generate_coverage_report:
-        python_utils.PRINT(
-            'Checking whether coverage is installed in %s'
-            % common.OPPIA_TOOLS_DIR)
-        if not os.path.exists(
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4')):
-            python_utils.PRINT('Installing coverage')
-            install_third_party_libs.pip_install(
-                'coverage', '4.5.4',
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4'))
+        python_utils.PRINT('Checking whether coverage is installed')
+        if subprocess.call(['which', 'coverage']) != 0:
```

Comment:
Can we use `python -m pip list` to check? Windows does not have this `which` command.

Reply:
Completely forgot about that, thanks for bringing that up! I also found that `pip show` is better since I don't have to parse through the output for the package.